runtime/phala/src/lib.rs

```diff
@@ -0,0 +1,1214 @@
+                c,
+                Call::System { .. }
+                    | Call::Timestamp { .. }
+                    | Call::Session { .. }
+                    | Call::Democracy { .. }
+                    | Call::Council { .. }
+                    | Call::PhragmenElection { .. }
+                    | Call::TechnicalCommittee { .. }
+                    | Call::TechnicalMembership { .. }
+                    | Call::Treasury { .. }
+                    | Call::Bounties { .. }
+                    | Call::Tips { .. }
+                    | Call::Utility { .. }
+                    | Call::Identity { .. }
+                    | Call::Vesting(pallet_vesting::Call::vest { .. })
+                    | Call::Vesting(pallet_vesting::Call::vest_other { .. })
+                    | Call::Scheduler { .. }
+                    | Call::Proxy { .. }
+                    | Call::Multisig { .. }
+            ),
+            ProxyType::CancelProxy => matches!(
+                c,
+                Call::Proxy(pallet_proxy::Call::reject_announcement { .. })
+                    | Call::Utility { .. }
+                    | Call::Multisig { .. }
+            ),
+            ProxyType::Governance => matches!(
+                c,
+                Call::Democracy { .. }
+                    | Call::PhragmenElection { .. }
+                    | Call::Council { .. }
+                    | Call::TechnicalCommittee { .. }
+                    | Call::Treasury { .. }
+                    | Call::Utility { .. }
+                    | Call::Bounties { .. }
+                    | Call::Tips { .. }
+                    | Call::Lottery { .. }
+            ),
+            ProxyType::Collator => matches!(
+                c,
+                Call::CollatorSelection { .. } | Call::Utility { .. } | Call::Multisig { .. }
+            ),
+        }
+    }
+    fn is_superset(&self, o: &Self) -> bool {
+        match (self, o) {
+            (x, y) if x == y => true,
+            (ProxyType::Any, _) => true,
+            (_, ProxyType::Any) => false,
+            (ProxyType::NonTransfer, _) => true,
+            _ => false,
+        }
+    }
+}
+
+impl pallet_sudo::Config for Runtime {
+    type Call = Call;
+    type Event = Event;
+}
+
+impl pallet_proxy::Config for Runtime {
+    type Event = Event;
+    type Call = Call;
+    type Currency = Balances;
+    type ProxyType = ProxyType;
+    type ProxyDepositBase = ProxyDepositBase;
+    type ProxyDepositFactor = ProxyDepositFactor;
+    type MaxProxies = MaxProxies;
+    type WeightInfo = pallet_proxy::weights::SubstrateWeight<Runtime>;
+    type MaxPending = MaxPending;
+    type CallHasher = Hasher;
+    type AnnouncementDepositBase = AnnouncementDepositBase;
+    type AnnouncementDepositFactor = AnnouncementDepositFactor;
+}
+
+parameter_types! {
+    pub MaximumSchedulerWeight: Weight = Perbill::from_percent(80) *
+        RuntimeBlockWeights::get().max_block;
+    pub const MaxScheduledPerBlock: u32 = 50;
```

Comment:
Sounds too large? But if it's in sync, we should be fine

Reply:
same <https://github.com/Phala-Network/khala-parachain/blob/main/runtime/khala/src/lib.rs#L492>